Convert some uses of passive voice to active voice in Orleans docs (#45474)
dotnet/docs: docs/orleans/migration-guide.md

## Migration

> Due to changes in how Orleans identifies grains and streams, you cannot easily migrate existing applications using reminders, streams, or grain persistence to Orleans 7.0. We plan to incrementally offer a migration path for these applications.

Reviewer: ```suggestion
Due to changes in how Orleans identifies grains and streams, you cannot (currently) easily migrate existing applications using reminders, streams, or grain persistence to Orleans 7.0.
```

Reviewer: Another rule is not to talk about future plans, in case they never materialize.
